modules/ingress/main.tf

```diff
@@ -2,12 +2,20 @@ locals {
   group_name = var.name
 
   //dummy_path has to he specified by default otherwise ingress can't be built
-  dummy_path = [{
-    name        = "response-static"
-    port        = null
-    path        = "/Pto48SjdzKBclyL5"
-    static_port = "use-annotation"
-  }]
+  dummy_path = [
+    {
+      name        = "response-static"
+      port        = null
+      path        = "/Pto48SjdzKBclyL5"
+      static_port = "use-annotation"
+    },
+    {
+      name        = "ssl-redirect"
```

Comment:
can we have ssl redirect enabling under option config, so that we will enable/disable it?